cli-output/src/cli_output.rs

```diff
@@ -1648,7 +1648,21 @@ impl VerboseDisplay for CliAuthorizedVoters {}
 
 impl fmt::Display for CliAuthorizedVoters {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "{:?}", self.authorized_voters)
+        if let Some((_epoch, current_authorized_voter)) = self.authorized_voters.first_key_value() {
```

Comment:
"surely we can't resolve the curre... why the hell do we do all of this superfluous work for every vote?"